Library/Formula/chktex.rb

```diff
@@ -1,14 +1,16 @@
-require 'formula'
-
 class Chktex < Formula
-  homepage 'http://www.nongnu.org/chktex/'
-  url 'http://download.savannah.gnu.org/releases/chktex/chktex-1.7.2.tar.gz'
-  sha1 '85d9e9fbf3e89104966bd725f156c826d0b44dd9'
+  homepage "http://www.nongnu.org/chktex/"
+  url "http://download.savannah.gnu.org/releases/chktex/chktex-1.7.2.tar.gz"
+  sha256 "d7f37985e3a122990f2a29fe7cac5d1f31acb1e50035457ef7ceb07c30550158"
 
   depends_on :tex
 
   def install
     system "./configure", "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"chktex", "-W"
```

Comment:
Would be good to write out a `.tex` file here and try and test that? If you `grep` through other formulae you will find similar approaches.